src/Symfony/Component/SignalRegistry/SignalRegistry.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Symfony\Component\SignalRegistry;
+
+final class SignalRegistry implements SignalRegistryInterface
+{
+    private $signals = [];
+
+    public function __construct(bool $asynchronousMode = true)
```

Comment:
Since developers can't point out the exact place where to dispatch signal, `$asynchronousMode = false` is useless and we should get rid of this mode. What do you think?

Reply:
Sure :+1: